tests/Oryx.Integration.Tests/StorageAccountSanityTestBase.cs

```diff
@@ -356,5 +364,12 @@ private string GetDefaultVersion(string debianFlavor, params string[] platformPa
 
             return defaultVersion;
         }
+
+        private string GetKeyvaultSecretValue(string keyvaultUri, string secretName)
```

Comment:
We have 4 different implementations of the `GetKeyVaultSecretValue` method. Lets pull them all into a shared  `GetKeyVaultSecretValue` method that is defined in the `Oryx.Tests.Common` project and can be used in each location.

Reply:
I was thinking the same initially. Thanks for pointing it out.

Reply:
a common work item which will address the issues: https://devdiv.visualstudio.com/DevDiv/_boards/board/t/Oryx/Stories/?workitem=1848573